dev/benchmarks/test_apps/stocks/test/locale_test.dart

```diff
@@ -11,18 +11,10 @@ void main() {
 
   testWidgets('Changing locale', (WidgetTester tester) async {
     stocks.main();
-    await tester.idle(); // see https://github.com/flutter/flutter/issues/1865
     await tester.pump();
     // The initial test app's locale is "_", so we're seeing the fallback translation here.
     expect(find.text('MARKET'), findsOneWidget);
-    await tester.binding.setLocale('es', 'US');
-    await tester.idle();
-
-    // The Localizations widget has been built with the new locale. The
```

Comment:
I guess the reasoning above might apply to why we don't have to idle and pump here too.